src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: "Avenir-Book", sans-serif ;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy", sans-serif ;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+div {
+  display: block;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search__form-block {
+  width: 100%;
+  display: flex;
+  height: 70px;
+  margin: 20px 0;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 3px;
+}
+
+.search__form-input {
```

Comment:
remove the outline (it doesn't exist in the design):
![image](https://user-images.githubusercontent.com/86923089/216818088-b3a1c81b-2ec8-4868-bbf5-e461b177bdba.png)